### DISCUSSION TITLE
WBI creates exact duplicates when appending

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#69] [General]
Is that a bug?
See https://www.wikidata.org/wiki/Lexeme:L242708 for an example
If not I will implement a check for myself, but it seems like a bug to me, because I assume that WBI gets the statement append the new claim and upload. It should of course fail in some way if the claim is EXACTLY the same as a claim already present because duplicates have no value.

**Comment by LeMyst** [comment]
Hello,

Thank you for your report.

I tried to reproduce the issue without success.
I think that can be an issue with the accented characters.

Can you try to reproduce the issue on your side with and without accented characters?

Can you give me machine information on which the script is run:
- Operating system
- Language
- Python version

Thank you
